Packs/PAN-OS/Integrations/Panorama/Panorama.yml

```diff
@@ -1954,6 +1954,8 @@ script:
       isArray: true
       name: Users
       required: true
+    - description: Timeout value to automatically unregister the users (in seconds). Only applicable to PAN-OS 9.x and higher.
+      name: timeout
```

Comment:
The ```panorama-register-user-tag``` is deprecated (see line 1961). Therefore we want to encourage our customers to use the ```pan-os-register-user-tag``` instead.

You simply remove it and implement it only on ```pan-os-register-user-tag``` command.

Reply:
Agreed, removing.